### PR TITLE
More last minute fixes.

### PR DESCRIPTION
**Pull request by xendk**
Closes most Redmine issues.

### CODE REVIEW

File: fbs.info [hunk 1]
```diff
@@ -2,11 +2,14 @@ name = FBS
...
 core = 7.x
 package = Providers
+version = 7.x-1.0-dev
+project = fbs
 dependencies[] = date_popup
 dependencies[] = ding_provider
 dependencies[] = ding_reservation
 dependencies[] = ding_user
 dependencies[] = entity
+dependencies[] = fbs
```
Comment (cableman) [inline fbs.info]: FBS dependent on FBS?

File: fbs.info [hunk 2]
```diff
@@ -43,3 +44,4 @@ files[] = includes/classes/FBSLogDrupal.php
 files[] = includes/classes/FBSLogInterface.php
 files[] = includes/classes/FBSLogStdout.php
 files[] = includes/classes/FBS.php
+mtime = 1439987908
```
Comment (cableman) [inline fbs.info]: Please remove the mtime variable

File: fbs.info (hunk 1 above)
Comment (xendk) [inline fbs.info]: Ask features. I guess it gets added because the feature uses fields from the module, and Features isn't smart enough to notice.

**Comment by cableman** [comment]
Reviewed and approved

**Pull request merged**